apps/language_server/lib/language_server/providers/formatting.ex

```diff
@@ -56,7 +56,8 @@ defmodule ElixirLS.LanguageServer.Providers.Formatting do
     |> Stream.flat_map(fn glob ->
       [
         Path.join([project_dir, glob]),
-        Path.join([project_dir, "apps", "*", glob])
+        Path.join([project_dir, "apps", "*", glob]),
+        Path.join([Path.dirname(file), glob])
```

Comment:
Shouldn't it be added only when `Path.dirname(file)` differs from `project_dir`?

Reply:
You're right. It should check file are in the project dir before add to the globs.

Reply:
Fixed

Reply:
Please, see comments below